content/docs/interface/fdp/_index.md

```diff
@@ -25,7 +25,45 @@ fair push config.yaml
 
 - read (and validate) the *config.yaml* file
 - generate a working *config.yaml* file (see [Working example]({{% ref "/docs/interface/example1" %}}))
-  - globbing (`*` and `**` replaced with all matching objects, all components listed), specific version numbers, and any variables in `run_metadata:`, `register:`, `read:`, and `write:` are replaced with true values, *e.g.*
+  - globbing is used to interpret `*` and `**` as all matching objects, with all components listed, and the original string returned, *e.g.* if `real/data/1@v.0.0.1` and `real/data/thing/1@v.0.0.1` already exist in the registry, the user-written config:
```

Comment:
Given that we're officially using [semver](https://semver.org), we should probably stick with it and not have the "v." at the front:

```suggestion
  - globbing is used to interpret `*` and `**` as all matching objects, with all components listed, and the original string returned, *e.g.* if `real/data/1@0.0.1` and `real/data/thing/1@0.0.1` already exist in the registry, the user-written config:
```

- I'm also not sure where (if anywhere?) you mention the difference between `*` and `**`
- I'm not sure about listing all of the components. If I decide to add an extra component to this data product, how will I know what to `use:` when writing the new component if the existing components are all explicitly listed? For reading, it just seems extremely time-consuming.

Reply:
> I have a couple of other issues:
> 
> * In L70 - https://github.com/FAIRDataPipeline/FDP_hugo/blame/edit-config/content/docs/interface/fdp/_index.md#L70 - `PATCH` should be `CLI.PATCH`, or `CLI.MINOR` higher up, should `MINOR`
> * In L72 - https://github.com/FAIRDataPipeline/FDP_hugo/blame/edit-config/content/docs/interface/fdp/_index.md#L72 - I think this could be simpler - maybe just "version defaults to `CLI.PATCH`", with a separate line saying "If the data product doesn't already exist, version increments should count from `0.0.0`, so `CLI.PATCH` gives `0.0.1`, etc."

* An old typo! Good catch. I've kept the CLI prefix because you wanted to discuss its removal in the meeting today...
* I think my version is simpler? But will change it if you want.

Reply:
* regarding `dataproduct@v.0.1.0` I was trying to use shorthand. I think `dataproduct@0.1.0` is less intuitive so I'll write versions out explicitly.

Reply:
* asterisks - sorry, I assumed this was standard convention; added a description
* components - I agree! But this is an old comment... it might be worth checking the rest to make sure nothing else has slipped through and needs updating!

Reply:
I can't actually see where you've done any of this? I think something is going slightly wrong with how I'm viewing this...

Reply:
Yea, my bad. I didn't push yet. But I've now got merge conflicts from the commit you requested.